xbps-src

```diff
@@ -679,7 +679,7 @@ if [ "$XBPS_CCACHE" ]; then
     export CCACHE_DIR="$XBPS_HOSTDIR/ccache"
     # Avoid not using cached files just due to compiler mtime
     # changes when e.g. bootstrapping
-    export CCACHE_COMPILERCHECK=content CCACHE_COMPRESS=1
+    export CCACHE_COMPILERCHECK=content
```

Comment:
Not sure but something like:
```
    if [ -z "${XBPS_CCACHE_NOCOMPRESS+set}" ]; then
        export CCACHE_COMPRESS=1
    fi
```

with `XBPS_CCACHE_NOCOMPRESS` from config file.

Disable ccache compression is NOT nice.